frame/grandpa/src/lib.rs

```diff
@@ -377,6 +396,11 @@ mod weight_for {
 			// fetching set id -> session index mappings
 			.saturating_add(T::DbWeight::get().reads(2))
 	}
+
+	pub fn note_stalled<T: super::Trait>() -> Weight {
```

Comment:
This is called `note_stalled`, but is just seems to compute a weight and not do anything else

Reply:
(oh, I see this is in the `weight_for` module)

Reply:
It's simple enough that I could have declared it in the weight attribute itself, but since we already have the `weight_for` module I decided to put it there 🤷.